tests/query/highlights/julia/test.jl

```diff
@@ -0,0 +1,26 @@
+function load_data(::Symbol; ::Int) :: Tuple
+# <- keyword.function
+#        ^^^^^^^^^ function
```

Comment:
one `^` symbol is enough
need not to be 8 or 9 characters like this
Looks a bit too much for my eye :)

Reply:
sure I'll fix that...I like to be explicit tho 😛